app/frontend/src/lib/ui/components/filterGroup.js

```diff
@@ -5,11 +5,16 @@ export const CHECKBOX_CLASS_SELECTOR = 'govuk-checkboxes__input';
 export const CLOSE_ALL_TEXT = 'Close all';
 export const OPEN_ALL_TEXT = 'Open all';
 
-window.addEventListener('DOMContentLoaded', () => init('filter-group__container', 'moj-filter__tag', 'clear-filters-button', 'close-all-groups', 'mobile-filters-button'));
+window.addEventListener(
+  'DOMContentLoaded',
+  () => init('filter-group__container', 'moj-filter__tag', 'clear-filters-button', 'close-all-groups', 'mobile-filters-button', 'govuk-accordion__section-button'),
```

Comment:
This line, and 13 and 16, is very long and hence hard to read on GH (need to locate horizontal scroll to scroll right, then scroll back up again). Do we have length-linting for JS?

Reply:
Yes we do

Reply:
if readabilty is an issue (and its getting that way) as there have been quite a few args added since this was first done could have the init arguments as an object and use `init({ argKey1, argKey2... })` syntax in the init function definition